js/index.js

```diff
@@ -73,3 +73,42 @@ if (
 ) {
   document.getElementById("messages").style.visibility = "hidden";
 }
+const githubRequest = new XMLHttpRequest();
+githubRequest.open("GET", "https://api.github.com/users/klove2016/repos");
+
+githubRequest.send();
+
+githubRequest.onload = function () {
+  const repos = JSON.parse(githubRequest.responseText);
+
+  const projectSection = document.getElementById("projects");
+  const projectList = projectSection.querySelector("ul");
+  for (let i = 0; i < repos.length; i++) {
+    eachRepo = repos[i];
+    project = document.createElement("li");
+    const repoStrName = eachRepo.name.toString();
+    const repoNoDashName = repoStrName.replaceAll("-", " ");
+    const repoRealName = repoNoDashName.split(" ");
```

Comment:
You can string all these functions together instead of saving it multiple times along the way. Also, there are a couple of simplifications you can make here. For example, `name` is already a string, and you can split on `"-"` instead of converting `"-"` to `" "` then splitting on `" "`:

```suggestion
    const repoRealName = eachRepo.name.split("-");
```